src/Symfony/Component/Console/Tests/Fixtures/application_run2.txt

```diff
@@ -1,28 +1,26 @@
 Usage:
-  help [options] [--] [<command_name>]
+  list [options] [--] [<namespace>]
```

Comment:
I updated the fixture, because now, the `--help` display the help of the default command.